### PR TITLE
Update SChinese csv

### PR DESCRIPTION
**Pull request by xyjx001**

### CODE REVIEW

File: Resources/string.csv [hunk 1]
```diff
@@ -79,6 +79,7 @@ GM,GM,GM(Призрак),GM,GM(管理员),GM(遊戲大師)
 
 # 属性
 LastImpostor,Last Impostor,Последний Предатель,ラストインポスター,绝境者,絕境者
+LastImpostorKillCooldown,LastImpostorKillCooldown,,,绝境者击杀冷却时间
```
Comment (soukunsandesu) [inline Resources/string.csv:82]: not require

Comment (xyjx001) [reply Resources/string.csv:82]: If it's removed, the host setting would show an error message like<INVALID: LastImpostorKillCooldown>

Comment (soukunsandesu) [reply Resources/string.csv:82]: image
``LastImpostorKillCooldown``  isn't use now.

Comment (soukunsandesu) [reply Resources/string.csv:82]: sorry. I'll fix it.
Just a minute.
After fix merged, do ``fetch`` and ``pull``.

Comment (xyjx001) [reply Resources/string.csv]: Done

**Review by soukunsandesu — APPROVED** [review]

**Pull request merged**